.ci/pytorch/win-test-helpers/installation-helpers/install_oneapi_bundle.bat

```diff
@@ -0,0 +1,37 @@
+set ONEAPI_PARENT_DIR=C:\Program Files (x86)\Intel
+set INTEL_ONEAPI_PYTORCH_BUNDLE_URL=https://registrationcenter-download.intel.com/akdlm/IRC_NAS/5ca2021d-dd1a-4ab1-bd52-758fe63cf827/w_intel-for-pytorch-gpu-dev_p_0.5.2.19_offline.exe
+set INTEL_ONEAPI_PYTORCH_BUNDLE_DISPLAY_NAME=intel.oneapi.win.intel-for-pytorch-gpu-dev.product
+set INSTALL_FRESH_ONEAPI=1
+
+if not exist "%ONEAPI_PARENT_DIR%\oneAPI" (
+    set INSTALL_FRESH_ONEAPI=1
+)
+
+
+if "%INSTALL_FRESH_ONEAPI%"=="1" (
+    IF EXIST "%ONEAPI_PARENT_DIR%\oneAPI\Installer\installer.exe" (
```

Comment:
Will we compare the bundle version between plan to install and installed, when the versions are same, we will not remove and reinstall? If not, let's simply the script logic, if there is installed bundle, always remove it and install the specific bundle in this script.